importer/api/__init__.py

```diff
@@ -144,6 +142,16 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    if element.tag == "video":
```

Comment:
Yep. Will do that when there are more. The original issue is for all types; this is just the "proof of concept" branch for this.